Makefile

```diff
@@ -2,14 +2,16 @@
 PATH := $(PATH):$(HOME)/.local/bin
 
 deps:
-	sudo apt install python3.8 python3.8-venv libpython3.8-dev libpq-dev graphicsmagick
+	sudo apt update
+	sudo apt install -y python3.8 python3.8-venv libpython3.8-dev libpq-dev graphicsmagick
 	# Install node: https://github.com/nodesource/distributions/blob/master/README.md#deb		
 	curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
 	sudo apt-get install -y nodejs
 	# Install poetry: https://python-poetry.org/docs/master/#osx--linux--bashonwindows-install-instructions 
 	curl -sSL https://install.python-poetry.org | python3.8 -	
 	sudo npm install -g yarn
-	sudo apt install postgresql
+	sudo apt install -y postgresql
+	sudo service postgresql start
```

Comment:
This is a good nudge, I forgot we hadn't been starting the psql service. My only gripe: I would recommend replacing `service` with `systemctl start postgresql`, `service` is a much older way of doing things and may become deprecated in the future.